core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/sql/SQLSourceOpExec.scala

```diff
@@ -186,7 +186,7 @@ abstract class SQLSourceOpExec(
       breakable {
         val columnName = attr.getName
         val columnType = attr.getType
-        val value = curResultSet.get.getString(columnName)
+        val value = curResultSet.get.getObject(columnName)
```

Comment:
any reason why you need to get the object and not a string?

Reply:
the old way of getting it as a string and parse is a bad design anyway. For example in this case we are in fact getting a `java.util.Boolean` but we read it as a string. So the `false` value would be read as a string `f`. Then `f` can't be parsed to `AttributeType.BOOLEAN`.